apps/fluent-tester/src/E2E/FocusZone/pages/FocusZonePageObject.win.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  FOCUSZONE_TESTPAGE,
+  FOCUSZONE_TEST_COMPONENT,
+  HOMEPAGE_FOCUSZONE_BUTTON,
+} from '../../../FluentTester/TestComponents/FocusZone/consts';
+import { BasePage, By } from '../../common/BasePage';
+
+class FocusZonePageObject extends BasePage {
+  // OVERRIDE: We use isExisting() here instead of isDisplayed() because FocusTrapZone does not have any UI to it, it's simply
```

Comment:
nit: change FocusTrapZone to FocusZone in comment

Reply:
Good catch, changing now